src/IBM.Watson.TextToSpeech.v1/Websockets/AWebSocketClient.cs

```diff
@@ -0,0 +1,188 @@
+﻿/**
+* Copyright 2017 IBM Corp. All Rights Reserved.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*
+*/
+
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Net.WebSockets;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace IBM.Watson.TextToSpeech.v1.Websockets
+{
+    public abstract class AWebSocketClient
+    {
+        public const string BINARY_STREAMS = "binary_streams";
+        public const string CONTENT_TYPE = "content_type";
+        public const string MARKS = "marks";
+        public const string WORDS = "words";
```

Comment:
fyi constants in .net should be in PascalCase (local constants should be camelCase). I dont care so much, esp since i think we are generating constants in upper camel. Just something to keep in mind!